packages/jitar-serialization/src/serializers/DateSerializer.ts

```diff
@@ -0,0 +1,39 @@
+
+import ValueSerializer from '../ValueSerializer.js';
+import SerializedDate from '../types/serialized/SerializedDate.js';
+import InvalidDateString from './errors/InvalidDateString.js';
+
+export default class DateSerializer extends ValueSerializer
+{
+    canSerialize(value: unknown): boolean
+    {
+        return value instanceof Date;
+    }
+
+    canDeserialize(value: unknown): boolean
+    {
+        const date = value as SerializedDate;
+        
+        return date instanceof Object
+            && date.serialized === true
+            && date.name === 'Date'
+            && typeof date.value === 'string';
+    }
+    
+    async serialize(date: Date): Promise<SerializedDate>
+    {
+        return { serialized: true, name: 'Date', value: date.toISOString() };
+    }
+
+    async deserialize(object: SerializedDate): Promise<Date>
+    {
+        const date = new Date(object.value);
```

Comment:
The current implementation doesn't accept null values because the `canDeserialize(value)` function checks if the value is a string type. The serialization process always provides a string value, so the question is if we want to support this for external clients the use our RPC API.